### DISCUSSION TITLE
جلسه سوم دوره بهترین مسیر ورود به دنیای برنامه‌نویسی

### DISCUSSION THREAD

**Discussion by alimd** [AliMD/alwatr#174] [Webinar & Meetups]
# جلسه سوم دوره بهترین مسیر ورود به دنیای برنامه‌نویسی

`در تاریخ: ۵-۵-۱۴۰۱`

### سلام و عرض ادب، خیلی خوش‌آمدید

لطفا اگر مطلب مفیدی در رستای محتوای مطرح شده دارید همراه با تمرین‌های این جلسه در اینجا مطرح کنید.

**مباحت مهم مطرح شده:**

۱.  نکاتی در مورد یادگیری بهتر و تاکید بر خود انگیزشی شخصی و همت زیاد

۲. مباحث پایه شبک‌های کامپیوتری و اینترنت

۳. مفهوم وب و تفاوت آن با اینترنت، مرورگر و ...

۴. شروع و معرفی HTML, XHTML, HTML 5.2 و مفاهیم پایه

۵. معرفی وب اپلیکیشن به عنوان هدف و آینده مباحث این دوره

۶. معرفی Progressive Web Application 
نمونه: https://github.com/hemanth/awesome-pwa#apps

۷. معرفی hybrid web applications
خیلی از اپلیکیشن‌های معروفی که روی گوشی نصب می‌کنید با تکنولوژی‌های وب ساخته شده‌اند مثلا
Twitter, Gmail, Apple App Store, Apple iBooks, Instagram, Evernote, Cryptochange, JustWatch, Untappd
برنامه‌های زیادی همچون vscode

**نکته مهم:**
میدونم این جمله باعث ناراحتی خیلی از دوستان میشه ولی متاسفانه واقعت تلخی هست که باید گفت!
معمولا هیچ کدام از این برنامه‌های پرسرعت با ابزارهای کندی مثل react, jquery, bootstrap, php, laravel و حتی انگولار و امثال اون نوشته نمی‌شود پس از الان مراقب باشید گول کامونیتی و عموم برنامه‌نویسان تازه‌کار رو نخورید!

[مطالب بیشتر در کانال تلگرام](https://t.me/AliMDSchool)

**Comment by alimd** [comment]
لیست دوستان شرکت کننده:

(چجوری آی‌دی همه رو جمع کنم وقتی همه پیام نمیدید)

**Reply by adltalab** [reply]
Younes Adltalab
[My Github](https://github.com/adltalab)

**Reply by A-sina** [reply]
[user](https://github.com/a-sina)
Sina asadi

**Reply by amirmmolazemi** [reply]
salam o dorood 
man amirmohammad molazemi hastam
[my github user](https://github.com/Amirmmolazemi)

**Reply by berenjian** [reply]
پیام

**Comment by A-sina** [comment]
salam va dorood

**Comment by alimd** [comment]
به شخصه برای من این PWA خیلی جذاب بود
https://2048.love2dev.com/

برای شما کدوم برنامه جذاب هستش؟

**Reply by A-sina** [reply]
In bazi ke besiar jazab va fun bood 
Alave bar in baraye bande calcuator ha va emojilog ham jaleb bood 
YT music o Spotify o Soundcload o PWA haye google ham ke jaye khod darand

**Reply by amirmmolazemi** [reply]
shayad behtarin PWA az nazar man mitoone telegram va spotify bashe ke vaghean karbordi va shayad be kheili az maha komak karde bashe.

**Reply by ghost** [reply]
Man chon vaghti be level bala mirsidam va misokhtam va lajam dar miaamad khily az in bazy khosham nmiad

**Reply by alimd** [reply]
لینک بدید لطفا که ماهم تست کنیم

**Comment by A-sina** [comment]
# tamrin jalase sevom 
salam va dorood mojadad
[tamrin](https://output.jsbin.com/jageyon)

**Comment by ghost** [comment]
Hello
May name is kazem moghadasan nishabory

**Comment by amirmmolazemi** [comment]
# salam ostad MihanDoost
### taklif jalase sevom https://output.jsbin.com/sisuqir/

**Comment by mostafahaghi** [comment]
مثل همیشه عالی

**Comment by mostafahaghi** [comment]
منابع HTML که برای شروع مناسب بوند اینجا میزارم:

https://www.w3schools.com

https://css-tricks.ir

**Reply by alimd** [reply]
تنها منبع مناسب همون MDN هستش ولی اینا برای شروع و فقط همون چند مطلب که معرفی شده توصیه میشه
مخصوصا w3school که ایرادات فراوانی داره ولی برای شروع خیلی خوب و روان توضیح داده

**Comment by sabermanesh** [comment]
سلام
[امین صابرمنش](https://github.com/sabermanesh)
مطلب مارک داون و **ایجاد عکس** و **لینک** در محیط
[تکلیف جلسه دوم](https://github.com/sabermanesh/AminMD)
مطلب سایت [w3school](https://www.w3schools.com) تا قسمت [HTML Formatting](https://www.w3schools.com/html/html_formatting.asp)
[تکلیف جلسه سوم](https://output.jsbin.com/vusobev)
@AliMD

**Comment by mahshidhooshmand** [comment]
سلام
[تمرین جلسه دوم(عکس و لینک)](https://github.com/mahshidhooshmand/my-second-repo)
[تمرین جلسه سوم(W3S)](https://jsbin.com/hijayoq)

**Comment by mahyasbm** [comment]
سلام 
محیا صابرمنش هستم.
مشق جلسه دوم: 

[مشق جلسه دوم 
](https://github.com/mahyasbm/mohajer)
[مشق جلسه سوم](https://output.jsbin.com/jesideb)

**Comment by Rahimpanah** [comment]
Hello teacher,
This my homework for session 3:
https://jsbin.com/kudizib

**Comment by amir91179** [comment]
سلام خسته نباشید
تمرین جلسه سوم
https://jsbin.com/vacexiv

**Comment by adltalab** [comment]
#### `Homework- session 3`
- [x] [github-page](#discussioncomment-3265035) <sub> *Done* :+1: </sub>
- [x]  [jsbin - w3school](https://jsbin.com/quqisuw)  <sub> *Done.* </sub>

**Comment by shosseinfarjadi** [comment]
salam

**Comment by taha-agh** [comment]
سلام
وقتی گوگل ترنسلیت رو به عنوان اکستنشن انتخاب میکنم چرا بعد انتخاب کلمه اون پایین علامت گوگل نمیاد باید رایت کلیک کنم تازه بعد که رو گوگل ترنسلیت میزنم مستقسم معنی رو نمیده باید برم تو سایت؟کسی میتونه راهنمایی کنه ؟؟؟ممنون

**Reply by adltalab** [reply]
شاید در تنظیمات افزونه، گزینه ی don't display icon or pop-up انتخاب شده.

**Comment by marihooshmand** [comment]
[تمرین جلسه دوم](https://output.jsbin.com/hijaway)

**Comment by MohammadHoseynSeyedi85** [comment]
سلام یک سوال من الان باید تا پایان مبحث color or comments سایت w3schools مطالعه کنم بعد یک پروژه بدم یا نه ؟ چون الان یک دونه پروژه که حاصل مطالعه تا پایان مبحث basics بود زدم و در زیر براتون ارسال میکنم

file:///C:/Users/user/Downloads/Desktop/test%20google

**Reply by MohammadHoseynSeyedi85** [reply]
البته این لینک کد هست خروجی رو براتون در پیوی فرستادم

**Comment by SMsajjadSM** [comment]
first try with mrkdown
https://github.com/SMsajjadSM/sajjadrepository
tnx

**Comment by AlirezaKhodayari** [comment]
Hello!!!
Exercise
https://jsbin.com/qelolul
